ct/outline.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/burgerga/ProxmoxVE/add_outline/misc/build.func)
+# Copyright (c) 2024 community-scripts ORG
+# Author: Gerhard Burger (burgerga)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+function header_info {
+clear
+cat <<"EOF"
+   ____        __  ___
+  / __ \__  __/ /_/ (_)___  ___
+ / / / / / / / __/ / / __ \/ _ \
+/ /_/ / /_/ / /_/ / / / / /  __/
+\____/\__,_/\__/_/_/_/ /_/\___/
+
+EOF
+}
+header_info
+echo -e "Loading..."
+APP="Outline"
+var_disk="8"
+var_cpu="2"
+var_ram="4096"
+var_os="debian"
+var_version="12"
+variables
+color
+catch_errors
+
+function default_settings() {
+  CT_TYPE="1"
+  PW=""
+  CT_ID=$NEXTID
+  HN=$NSAPP
+  DISK_SIZE="$var_disk"
+  CORE_COUNT="$var_cpu"
+  RAM_SIZE="$var_ram"
+  BRG="vmbr0"
+  NET="dhcp"
+  GATE=""
+  APT_CACHER=""
+  APT_CACHER_IP=""
+  DISABLEIP6="no"
+  MTU=""
+  SD=""
+  NS=""
+  MAC=""
+  VLAN=""
+  SSH="no"
+  VERB="no"
+  echo_default
+}
+
+function update_script() {
+header_info
+check_container_storage
+check_container_resources
+if [[ ! -d /opt/outline ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
+RELEASE=$(curl -s https://api.github.com/repos/outline/outline/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
```

Comment:
I copied this from https://github.com/community-scripts/ProxmoxVE/blob/fab35a61902384a1425f6211a6511f489cc42d69/ct/linkwarden.sh#L60

Which is why I think https://github.com/community-scripts/ProxmoxVE/pull/330#issuecomment-2495668159 is a good idea, with maybe also a function to extract a semver without `v`?

Reply:
Unfortunately, we don't have a real standard at the moment, as all the repos in GitHub are always different in some way.

Reply:
Yes unfortunately it is not consistent at the moment. I think we always should keep the `v` prefix outside of the variable because it is not part of the semantic version and it is not clear if a prefix is included or not when it is not explicitly shown in the code. Either I have to check the external github url to know, or in this case I understood that it has a `v` prefix because of this line `outline-${RELEASE:1}`. Better to show prefixes explicitly and only keep the semantic version in the variable imo.